src/styles/blocks/card.scss

```diff
@@ -0,0 +1,91 @@
+.product-card {
+  width: 200px;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  background-color: #fff;
```

Comment:
The background color should be defined using a SCSS variable. Ensure that all main values are stored in a separate variables file as per the task requirements.